source-reminder/Makefile

```diff
@@ -0,0 +1,10 @@
+deploy: install-deps
+	zip -r deployment_package *
+	aws lambda update-function-code \
+	--region us-east-1 \
+		--function-name sourceReminder \
+		--zip-file fileb://deployment_package.zip
+	rm deployment_package.zip
+
+install-deps:
+	pip3 install -r $$(pwd) requirements.txt
```

Comment:
Forgive my python ignorance, but is `$$(pwd) requirements.txt` supposed to be a path?
I got this, but maybe I'm doing something wrong.
![Screen Shot 2021-07-20 at 10 26 19 AM](https://user-images.githubusercontent.com/2974713/126351669-9e94b629-0a36-4079-a5ad-bc07c7478738.png)

`pip3 install -r $$(pwd)/requirements.txt` seems to work

Reply:
Confirming `pip3 install -r $$(pwd)/requirements.txt` works for me, too.